Sources/SmileID/Classes/RectangleDetector/Quadilateral.swift

```diff
@@ -0,0 +1,284 @@
+//  Created by Boris Emorine on 2/9/18.
+//  Copyright © 2018 WeTransfer. All rights reserved.
```

Comment:
For each file in `RecantangleDetector`, can you include a comment with a link to where this file was obtained from?